_data/ballerinaopenapisupportswanlake.yml

```
@@ -0,0 +1,28 @@
+- title: Ballerina OpenAPI Support
+  url: #
+  sublinks:
+    - title: Using the Capabilities of the OpenAPI Tools
+      url: /learn/ballerina-openapi-support/#using-the-capabilities-of-the-openapi-tools
+      active: using-the-capabilities-of-the-openapi-tools
+      sublinks:
+        - title: Generating Ballerina Service from the OpenAPI Definition (Design-First Approach)
```

```suggestion
        - title: Generating a Ballerina Service from the OpenAPI Definition (Design-First Approach)
```
Shall we remove the part in brackets from the title? We cannot include that in the URL anyway.

---

We are adding these brackets with the purpose of emphasizing the user to our approach with standard API design approaches. I add this to the URL this way `generating-ballerina-service-from-the-openapi-definition-design-first-approach` then it work in local host. 😃